docs/development.rst

```diff
@@ -180,14 +198,17 @@ Client Side Testing
 -------------------
 
 Using the same setup as above for the Server Side Tests, your environment will be set up
-to run client side tests. Running ::
+to run client side tests. Running client side is the same as running server side test. Run ::
 
     cd girder-build
     ctest
 
 will run all of the tests, which include the client side tests.  Our client tests use the
 Jasmine JS testing framework.
 
+If you encounter errors regarding ESLINT or PUG_LINT, there is a chance you missed certain steps for setting up development dependencies.
+You could use ``ccmake`` to change ``CMake`` configuration. Or, it might be easier to recreate the environment from the beginning.
```

Comment:
The solution to such errors is likely to run `girder-install web --dev` followed by running `cmake` on the build directory, so we should prescribe that here.